SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2213,24 +2213,25 @@ FRPCErrorInfo USpatialReceiver::ApplyRPCInternal(UObject* TargetObject, UFunctio
 		}
 		else
 		{
-			if (EventTracer->IsEnabled() && RPCType != ERPCType::CrossServer)
+			uint64 RPCId = RPCService->GetLastAckedRPCId(EntityId, RPCType) + 1;
+			bool bUseEventTracer = EventTracer->IsEnabled() && RPCType != ERPCType::CrossServer && RPCId != 0;
```

Comment:
I would really like some templating/preprocessor magic for EventTracing ifs.
How did the NFR tests go?

Reply:
NFRs where fine when ET is turned off it seems. I agree preprocessor would be good! Perhaps a different PR though. Will make a ticket.